app/presenters/hyrax/fixity_status_presenter.rb

```diff
@@ -34,7 +34,6 @@ def render_file_set_status
 
       # A weird display, cause we need it to fit in that 180px column on
       # FileSet show, and have no real UI to link to for files/versions :(
-      # rubocop:disable Metrics/MethodLength
```

Comment:
Is there a reason you're removing these and moving to the global?  These are preferred as it scopes the problem to a few lines rather than the whole file.